pkg/test/constants/constants.go

```diff
@@ -30,7 +30,7 @@ var TestNameRegexesToSkip = []string{
 	// TODO(b/220357089): re-enable eventfunction test in long-running CRUD test suite.
 	".*(eventfunction).*",
 	// TODO(b/215781076): re-enable gke hub tests in long-running CRUD test suite.
-	".*(gkehubfeaturemembership|mcifeature).*",
+	".*(mcifeature).*",
```

Comment:
Do we know how long the `gkehubfeaturemembership` tests take? The comment says it needs to be added to long-running.

Reply:
It will take ~20 min as the test creates a gke cluster